qCC/ccQCustomPlot.h

```diff
@@ -126,14 +125,15 @@ class QCPColoredBars : public QCPBars
 			if (colors.size() > i)
 				newData.color = colors[i];
 			m_coloredData.insertMulti(newData.key, newData);
-			mData->insertMulti(newData.key, newData);
+//			data()->insertMulti(newData.key, newData);
+			QCPBars::addData(newData.key,newData.value);
```

Comment:
This is the main thing I wasn't 100% sure of. `insertMulti` doesn't exist, and I'm now only adding the double, not the struct - and I'm doing it through the QCPBars class, not the data().